app/client/public/index.html

```diff
@@ -109,7 +116,7 @@
 		<meta
 			prefix="og: http://ogp.me/ns#"
 			property="og:image"
-			content="%PUBLIC_URL%/litmus.png"
+			content="https://hub.litmuschaos.io/icons/litmus.png"
```

Comment:
This is not the correct way, How will you manage this in staging or development mode

Reply:
https://socialsharepreview.com/?url=https://staging-hub.litmuschaos.io/
If you check this site. It's not able to catch the %PUBLIC% thing....even for twitter card documentation they need an absolute path...so we either have to host it somewhere and give the link...or use this method where it always takes the image from the hub.
Personally I'm not a fan of this approach either, but mayadata.io website does the same thing
https://github.com/mayadata-io/website-mayadata/blob/staging/src/index.html

The best option, according to me, is to host the images somewhere for all websites. @chandankumar4

Reply:
@arkajyotiMukherjee you can try using `%PUBLIC_URL%/icons/litmus.png` `%PUBLIC_URL` resolves to the root of the website and that's the reason it is not showing up because the image is in `/icons`

Reply:
It was not in icons previously, it was on the root itself...check the old code... I've put it in icons for this PR

Reply:
Can you try `content="icons/litmus.png"` ?